Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/IPay88TokenizationViewModel.swift

```diff
@@ -0,0 +1,547 @@
+//
+//  IPay88TokenizationViewModel.swift
+//  PrimerSDK
+//
+//  Created by Evangelos on 12/12/22.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+import UIKit
+
+#if canImport(PrimerIPay88SDK)
+import PrimerIPay88SDK
+#endif
+
+class IPay88TokenizationViewModel: PaymentMethodTokenizationViewModel {
+    
+    deinit {
+        log(logLevel: .debug, message: "🧨 deinit: \(self.self) \(Unmanaged.passUnretained(self).toOpaque())")
+    }
+    
+#if canImport(PrimerIPay88SDK)
+    private var backendCallbackUrl: URL!
+    private var primerTransactionId: String!
+    private var statusUrl: URL!
+    private var resumeToken: String!
+    private var primerIPay88ViewController: PrimerIPay88ViewController!
+    private var primerIPay88Payment: PrimerIPay88Payment!
+    private var didCancel: (() -> Void)?
+    private var didComplete: (() -> Void)?
+    private var didFail: ((_ err: PrimerError) -> Void)?
+#endif
+    
+    private lazy var iPay88NumberFormatter: NumberFormatter = {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        numberFormatter.decimalSeparator = "."
+        numberFormatter.groupingSeparator = ","
```

Comment:
Are we 100% sure this is not tied to any locale?

Reply:
`iPay88` has no docs on this, I tried with a couple of combinations. Never managed to get the `groupingSeparator` cause they don't allow payments in sandbox over 999.99, they throw an `Transaction overlimit` error. @semirp anything on Android?

Reply:
They just say we should use 1,234.55 format @EvansPie